octoprint_mrbeam/migrate.py

```diff
@@ -627,6 +645,21 @@ def disable_gcode_auto_deletion(self):
         if not self.plugin.isFirstRun():
             self.plugin._settings.set_boolean(["gcodeAutoDeletion"], False)
 
+    def fix_s_series_mount_manager(self):
+        """
+        fixes a problem with the images before 25.6.2021
+        the rc.local file was missing the clear command for the mount_manager
+        this replaces the rc.local file with the one containing this row
+        """
+        self._logger.info("start fix_s_series_mount_manager")
+        src = os.path.join(
+            __package_path__, self.MIGRATE_FILES_FOLDER, "fix_s_series_mount_manager"
+        )
+        dst = "/etc/rc.local"
+        exec_cmd("sudo cp {src} {dst}".format(src=src, dst=dst))
```

Comment:
get the return value (Boolean) and add it to the following line (logging), so that the logged information is always correct.

Reply:
Or have it as an if statement
```python
if(exec_cmd( ... )):
  log("Great")
else:
  log("not so great")

Reply:
In fact, `exec_cmd()` already logs an error when the exit code is not `0` so I don't think this is necessary

Reply:
I did it with the if else